src/main/java/com/sngular/kloadgen/processor/util/SchemaProcessorUtils.java

```diff
@@ -241,20 +272,20 @@ private static MessageDefinition buildProtoMessageDefinition(
     final MessageDefinition.Builder msgDef = MessageDefinition.newBuilder(fieldName);
     final var element = (MessageElement) messageElement;
     extracted(globalNestedTypesByLevelAndMessage, msgDef, element.getFields(), nextDeepLevel, fieldName);
-    for (var optionalField : element.getOneOfs()) {
+    for (final var optionalField : element.getOneOfs()) {
       extracted(globalNestedTypesByLevelAndMessage, msgDef, optionalField.getFields(), nextDeepLevel, fieldName);
     }
     return msgDef.build();
   }
 
   private static void extracted(
-      final HashMap<Integer, HashMap<String, HashMap<String, TypeElement>>> globalNestedTypesByLevelAndMessage, final Builder msgDef, final List<FieldElement> fieldElementList,
-      final int deepLevel, final String messageName) {
+    final HashMap<Integer, HashMap<String, HashMap<String, TypeElement>>> globalNestedTypesByLevelAndMessage, final Builder msgDef, final List<FieldElement> fieldElementList,
+    final int deepLevel, final String messageName) {
 
     final HashMap<String, TypeElement> nestedTypes = processLevelTypes(globalNestedTypesByLevelAndMessage, msgDef, fieldElementList, deepLevel,
-                                                                       messageName);
+      messageName);
```

Comment:
Are you sure your autoformatter is set to use the OSS style?

Reply:
I have just added correctly formatter. It should already look fine.